types/cookiebot-sdk/index.d.ts

```diff
@@ -0,0 +1,41 @@
+// Type definitions for Cookiebot SDK
```

Comment:
This line should read `// Type definitions for non-npm package Cookiebot SDK` (and possibly a version number to which these typings refer).